src/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from './types';
+
+const findParent = (
+  person: Person,
+  parentType: string,
+  name: string,
+  peopleList: Person[],
```

Comment:
We can make typing look a little better by bringing the types into a separate interface or type.